packages/fa-icon-chooser/src/components/fa-icon-chooser/fa-icon-chooser.tsx

```diff
@@ -310,6 +320,14 @@ export class FaIconChooser {
 
     const kit = get(response, 'data.me.kit');
     this.kitMetadata = kit;
+
+    const embedProSvg = get(kit, 'permits.embedProSvg', []);
+
+    if (embedProSvg.length === 0) {
+      // return early so we do not update the familyStyles to include all styles in the release.familyStyles object
+      return;
+    }
```

Comment:
Here is the "yucky" part of this whole thing, right here.
We do NOT want the familyStyles to continue to build if we DO have a free, pro-lite, or pro+ lite kit.
So we need to short circuit this function and return before that is done.

Reply:
@robmadole ---

I don't think so, since iconUploads are for PRO and PRO PLUS accounts not lite and free accounts.
Unless I am missing something... I see no issues and ALL manual testing has come out just fine.